docs/advanced-guides/cloud-in-a-box.md

```diff
@@ -73,15 +74,71 @@ system release changes.  The installation of historical releases is currently no
 6. The deployment will start. This takes some time and the system will shutdown when the
    deployment is finished. This takes roughly an hour, possibly longer depending on the
    hardware and internet connection.
 
-7. Start the system again. System is ready for use, by default DHCP is tried on the first network device.
+### Manual installation
+
+:::note
+Using the autoinstall above is recommended.
+:::
 
-8. Login via SSH. Use the user `dragon` with the password `password`.
-   (You can obtain the ip address by inspecting the logs of your dhcp server or from the *issue text* of the virtual consoles of the system)
+1. Follow the [manual provisioning guide](../guides/deploy-guide/bootstrap.md#manual-provisioning), skip the part about
+   disk layout, and see below how to partition your drive.
+2. Drive partitioning - The drive should be at least 1 TB big, and this is because part of the drive is used to create 6
+   ceph OSDs.
+   ![Disk layout](images/cloud-in-a-box/disk-layout.png)
+    1. If you install on bare-metal, you can follow the manual provisioning guide mentioned above for UEFI.
+    2. Create a 1 GB ext4 partition mounted in `/boot`
+    3. Create an 8 GB swap partition.
+    4. Create a 120 GB unformatted partition.
+    5. Use a "Create volume group (LVM)" to create a volume group called `system` with the size of 120 GB on the
+       partition 4 you just created.
+    6. Create a logical volume by selecting the "Free Space" option under `system` LVM. This volume should be mounted in
+       `/` and have size of 100 GB.
+    7. Create a partition the size of the rest of the drive's space.
+    8. Create a new LVM volume group on partition 5 called `osd-vg`. This will be used for ceph.
+3. After the Ubuntu installation, the system will be rebooted.
+4. Log into the machine via console to get its IP and then use SSH to connect to the machine.
+5. Clone the [`cloud-in-a-box`](https://github.com/osism/cloud-in-a-box) repository into `/opt/cloud-in-a-box`
+   ```bash
+   sudo git clone https://github.com/osism/cloud-in-a-box /opt/cloud-in-a-box
+   ```
+6. Disable conflicting services from the default Ubuntu install.
+   ```bash
+   sudo /opt/cloud-in-a-box/cleanup.sh
+   ```
+7. Update system. At least an `apt update` is needed for the following scripts.
+   ```bash
+   sudo apt update
+   sudo apt upgrade
+   ```
+8. Run the `bootstrap` script with the required [type](#types).
+   ```bash
+   sudo /opt/cloud-in-a-box/bootstrap.sh sandbox
+   ```
+9. Run the `deploy` script with the required [type](#types) to install OpenStack
+   ```bash
+   sudo /opt/cloud-in-a-box/deploy.sh sandbox
+   ```
+10. Turn off the machine
+
+:::warning
+
+The installation scripts are not idempotent. In case there is any fail during `bootstrap.sh` or `deploy.sh` you have to
+start over with fresh installation.
+
+:::
+
+### Post instalation
+
+1. Start the system again. System is ready for use, by default DHCP is tried on the first network device.
```

Comment:
From my point of view the automated installation procedure is the usual way to get a CIAB installation and its a good thing to try to prevent duplication whenever there is a chance for it.
Nevertheless my opinion is, that we should not split the installation procedure in two separate sections because the users will tend not read/follow the remaining steps.
Duplication might be ok, but we can also reference the remaining steps by an ancor or simply by "...now continue from step 11 in the 'Automated installation' section".